games-puzzle/kpat/kpat-17.12.0.recipe

```diff
@@ -0,0 +1,140 @@
+SUMMARY="Solitaire Game by KDE"
+DESCRIPTION="KPat (aka KPatience) is a relaxing card sorting game. \
+To win the game a player has to arrange a single deck of cards \
+in certain order amongst each other. "
+HOMEPAGE="https://edu.kde.org/kpat"
+COPYRIGHT="2010-2017 KDE Organisation"
+LICENSE="GNU LGPL v2"
```

Comment:
There seems to be [a custom license][0] around as well. Please add it.

If this package also contains the documentations, they're subjected to [GNU FDL][1].

[0]: https://github.com/KDE/kpat/blob/v17.12.1/COPYING#L17
[1]: https://github.com/KDE/kpat/blob/v17.12.1/COPYING.DOC#L1